spec/lib/tasks/db_data_migrate_rake_spec.rb

```diff
@@ -141,5 +141,66 @@
         expect(openid_authentication_service).to be_a OpenidAuthenticationService
       }
     end
+
+    shared_examples 'a consistency migration' do |prep_method_sym|
+      let(:record_class) { record.class }
+      before do
+        expect(record).to be_persisted
+      end
+      context 'record is consistent' do
+        let(:prep_method) { send(prep_method_sym) }
+        before do
+          prep_method.call
+        end
+        it 'should update is_consistent to true' do
+          expect(record_class.where(is_consistent: nil)).to exist
+          invoke_task
+          expect(record_class.where(is_consistent: nil)).not_to exist
+          record.reload
+          expect(record.is_consistent).to eq true
+        end
+      end
+
+      context 'record is not consistent' do
+        it 'should update is_consistent to false' do
+          invoke_task
```

Comment:
Is there a reason that you're not checking `record_class.where(is_consistent: nil)` like you're doing in the above it block?